runbotics/runbotics-desktop/src/core/bpm/runtime/runtime.service.ts

```diff
@@ -148,6 +149,9 @@ export class RuntimeService implements OnApplicationBootstrap, OnModuleDestroy {
             ...(request.userId && { user: { id: request.userId } }),
             ...(request.callbackUrl && { callbackUrl: request.callbackUrl }),
         };
+
+        delete processInstance?.decryptedCredentials;
```

Comment:
function programming

Reply:
change to const { ...credentials, ...processInstanceWithoutCredentials } = processInstnace